http/cancel.go

```diff
@@ -0,0 +1,31 @@
+package http
+
+import (
+	"fmt"
+	"net/http"
+
+	"github.com/duneanalytics/duneapi-client-go/config"
+	"github.com/duneanalytics/duneapi-client-go/models"
+)
+
+var cancelURLTemplate = "https://%s/api/v1/execution/%s/cancel"
+
+func QueryCancel(env config.Env, executionID string) (*models.CancelResponse, error) {
```

Comment:
type QueryExecution interface {
   Cancel()
   Status()
   Result()
}

this becomes:

`func (e *QueryExecution) Cancel()`